src/apps/racks/models.py

```diff
@@ -0,0 +1,50 @@
+from sqlalchemy import (
+    DECIMAL,
+    Boolean,
+    Column,
+    DateTime,
+    ForeignKey,
+    Integer,
+    String,
+    Table,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.sqltypes import DateTime
+
+from src.core.utils.orm import (
+    default_available_rack_levels,
+    default_available_rack_weight,
+)
+from src.core.utils.time import get_current_time
+from src.core.utils.utils import generate_uuid
+from src.database.db_connection import Base
+
+
+class Rack(Base):
```

Comment:
new fields should be added - reserved_weight and weight_to_reserve, for displaying the weight reserved for rack levels